SBAT.md

```diff
@@ -213,68 +214,95 @@ product. Setting a product-specific generation number for such an event
 eliminates the need for other vendors to have to re-release the binaries for
 their products with an incremented global number.
 
-However, once the global number is bumped for the next upstream CVE fix there
-will be no further need to carry that product-specific generation number.
-Satisfying the check of the global number will also exclude any of the older
-product-specific binaries.
+The current consensus is for both generation numbers to only ever go up and not
```

Comment:
No need to say "current consensus" here IMHO. How about simply:

Both generation numbers should only ever go up; they should never be reset.